tsconfig.json

```diff
@@ -20,7 +20,7 @@
 	  "downlevelIteration": true,
 	  "skipLibCheck": true,
 	  "jsx": "react-jsx",
-	  "moduleResolution":"Node",
+	  "moduleResolution":"Bundler",
```

Comment:
This is needed to be able to use the new export from kaoto: `@kaoto/kaoto/models`